src/Apps/CollectorVerifcation/routes.tsx

```diff
@@ -0,0 +1,42 @@
+import { ErrorPage } from "Components/ErrorPage"
+import { RouteConfig } from "found"
+import React from "react"
+import { graphql } from "react-relay"
+import createLogger from "Utils/logger"
+import { CollectorVerificationApp } from "./CollectorVerificationApp"
+
+const logger = createLogger("Apps/CollectorVerification/routes")
+
+export const routes: RouteConfig[] = [
+  {
+    path: "/collector-verification/:id",
+    Component: CollectorVerificationApp,
```

Comment:
Be sure to code-split at all routes: https://github.com/artsy/reaction/blob/master/src/Apps/Conversation/routes.tsx#L10